docker/default.conf

```diff
@@ -44,6 +44,8 @@ server {
   rewrite ^/opensuse/(.*) https://pkg.jenkins.io/opensuse/$1 permanent;
   rewrite ^/debian/(.*) https://pkg.jenkins.io/debian/$1 permanent;
 
+  include includes/jira-to-github-redirects.conf;
+
```

Comment:
```suggestion
```

Restore initial content as issue-redirect.jenkins.io has been put in place.

Ref:
- https://github.com/jenkins-infra/helpdesk/issues/4889